docs/user/integrations/dna_center.md

```diff
@@ -0,0 +1,81 @@
+
+`*` As of SSoT 3.2.0 the LocationType for Areas, Buildings, and Floors can be defined in the Job form. Prior to SSoT 3.2.0 the DNA Center integration creates a Region, Site, and Floor LocationType and imports Areas as Region Locations, Buildings as Site Locations, and Floors as Floor Locations.
+
+`**` If the [Device Lifecycle Nautobot app](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt) is found to be installed, a matching Version will be created with a RelationshipAssociation connecting the Device and that Version.
+
+## Usage
+
+Once the app is installed and configured, you will be able to perform an inventory ingestion from DNA Center into Nautobot. From the Nautobot SSoT Dashboard view (`/plugins/ssot/`), DNA Center will show as a Data Source.
+
+![Dashboard View](../../images/dnac_dashboard.png)
+
+From the Dashboard, you can also view more information about the App by clicking on the `DNA Center to Nautobot` link and see the Detail view. This view will show the mappings of DNA Center objects to Nautobot objects, the sync history, and other configuration details for the App:
+
+![Detail View](../../images/dnac_detail-view.png)
+
+In order to utilize this integration you must first enable the Job. You can find the available installed Jobs under Jobs -> Jobs:
+
+![Job List](../../images/dnac_job_list.png)
+
+To enable the Job you must click on the orange pencil icon to the right of the `DNA Center to Nautobot` Job. You will be presented with the settings for the Job as shown below:
+
+![Job Settings](../../images/dnac_job_settings.png)
+
+You'll need to check the `Enabled` checkbox and then the `Update` button at the bottom of the page. You will then see that the play button next to the Job changes to blue and becomes functional, linking to the Job form.
+
+![Enabled Job](../../images/dnac_enabled_job.png)
+
+Once the Job is enabled, you'll need to manually create a few objects in Nautobot to use with the Job. First, you'll need to create a Secret that contains the username and password for authenticating to your desired DNA Center instance:
+
+![Username Secret](../../images/dnac_username_secret.png)
+
+![Password Secret](../../images/dnac_password_secret.png)
+
+Once the required Secrets are created, you'll need to create a SecretsGroup that pairs them together and defines the Access Type of HTTP(S) like shown below:
+
+![DNAC SecretsGroup](../../images/dnac_secretsgroup.png)
+
+With the SecretsGroup defined containing your instance credentials you'll then need to create an ExternalIntegration object to store the information about the DNA Center instance you wish to synchronize with.
+
+![DNAC ExternalIntegration](../../images/dnac_external_integration.png)
+
+> The only required portions are the Name, Remote URL, Verify SSL, and Secrets Group. The `Extra Config` section allows you to specify the port that DNA Center is running on. It will default to 443 if unspecified.
```

Comment:
IMO this would be easier to follow if the screenshot was split and this section of text comes before the "Advanced Configuration" panel screenshot

Reply:
So the LocationType that's used is actually up to the user as you have the option to specify what to use in the Job form. I've gone ahead and split the ExternalIntegration image as suggested.